fms/models/hf/gpt_bigcode/configuration_gpt_bigcode_hf.py

```diff
@@ -0,0 +1,82 @@
+from typing import Optional
+
+from transformers import PretrainedConfig
+
+from fms.models.gpt_bigcode import GPTBigCodeConfig
+
+
+class GPTBigCodeHFConfig(PretrainedConfig):
+    model_type = "gpt_bigcode_hf"
+
+    attribute_map = {
+        "vocab_size": "src_vocab_size",
+        "hidden_size": "emb_dim",
+        "num_attention_heads": "nheads",
+        "num_hidden_layers": "nlayers",
+    }
+
+    def __init__(
+        self,
+        src_vocab_size: Optional[int] = 49280,
+        emb_dim: Optional[int] = 2048,
+        emb_kq: Optional[int] = None,
+        emb_v: Optional[int] = None,
```

Comment:
is emb_v different at any point from emb_kq?

Reply:
We don't seem to be handling this param in the current code and just falling back to 

```python
self.config.emb_dim // self.config.nheads
```

for both the `emb_kq` and `emb_v`

I believe we should be doing:

```python
emb_v = (
     self.config.emb_dim // self.config.nheads
     if self.config.emb_v is None
     else self.config.emb_v
)
```

Reply:
removed these parameters as they are always fixed